crates/sui-move-build/src/lib.rs

```diff
@@ -215,6 +222,21 @@ impl BuildConfig {
     }
 }
 
+fn report_suppressed_linter_warnings(
+    filtered_diags_num: usize,
+    filtered_categories: usize,
+) -> anyhow::Result<()> {
+    if filtered_diags_num > 0 {
+        if let Err(err) = writeln!(
+            &mut std::io::stderr(),
+            "Total number of linter warnings suppressed: {filtered_diags_num} (filtered categories: {filtered_categories})\n"
```

Comment:
I went for succinctness here but I am happy to revisit what gets printed and how (should we perhaps introduce a verbose mode to build/test/publish/upgrade if we want more stats?).

Reply:
Do you have an example output? maybe we should have a test for this, or would that be too noisy?

Reply:
I added a CLI test for this.